pom.xml

```diff
@@ -184,27 +185,27 @@
         <version>${spring.version}</version>
       </dependency>
 
-      <dependency>
-        <groupId>javax.transaction</groupId>
-        <artifactId>javax.transaction-api</artifactId>
-        <version>1.2</version>
-      </dependency>
       <dependency>
         <groupId>javax.interceptor</groupId>
         <artifactId>javax.interceptor-api</artifactId>
-        <version>1.2</version>
+        <version>1.2.2</version>
+      </dependency>
+      <dependency>
+        <groupId>javax.transaction</groupId>
```

Comment:
You should use the new jakarta artifact here:
https://mvnrepository.com/artifact/jakarta.transaction/jakarta.transaction-api/1.3.2

Reply:
Yes, totally agree. Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.